src/index.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
```

Comment:
- [ ] Since you have already deleted the import for reportWebVitals, please also remove it from the file you are still calling it at the end as `reportWebVitals();` and we get a console error 
![image](https://user-images.githubusercontent.com/97521557/228337506-d57aec64-c249-4ba6-8b6f-428db333eae6.png)

- [ ] Please also remove the` App.test.js` and `setupTests.js` files as they are redundant.